### ISSUE TITLE
Wrong warning for visibleVariations prop type

### ISSUE THREAD

**Issue by me-chell** [pluslabeplus/vtex-io-intelisense#6]
I have this code: `"visibleVariations": ["talle"]` and the extension warns me `Incorrect type. Expected "object".` but on VTEX documentation its definitely `string` type.

Reference: [SKU Selector](https://developers.vtex.com/vtex-developer-docs/docs/vtex-store-components-skuselector)

**Comment by Luis-eplus** [comment]
Thanks for your report.

I see the SKU Selector documentation and they say that `visibleVariations` is of type `array`, after they show the props of `visibleVariations`, in this case, the prop `name` of type `string`, so, if documentations its not wrong,  this is the correct use of this porp:

```json
"visibleVariations": [
    { "name": "talle" }
]
```

![image](https://user-images.githubusercontent.com/86304045/151440281-5bbae45c-b2a4-4b85-9588-e3ce629491b4.png)

**Comment by me-chell** [comment]
Hi @Luis-eplus, yes, the docs are wrong. Seeing the typings on the app I see this interface. Taken from [here](https://github.com/vtex-apps/store-components/blob/master/react/components/SKUSelector/Wrapper.tsx)

```jsonc
interface Props {
  skuItems: ProductItem[]
  skuSelected: ProductItem | null
  onSKUSelected?: (skuId: string) => void
  maxItems?: number
  visibility?: string
  seeMoreLabel: string
  hideImpossibleCombinations?: boolean
  disableUnavailableSelectOptions?: boolean
  showValueNameForImageVariation?: boolean
  showValueForVariation?: ShowValueForVariation
  imageHeight?: ResponsiveValuesTypes.ResponsiveValue<number>
  imageWidth?: ResponsiveValuesTypes.ResponsiveValue<number>
  thumbnailImage?: string
  visibleVariations?: string[]
  showVariationsLabels?: ShowVariationsLabels
  variationsSpacing?: number
  showVariationsErrorMessage?: boolean
  initialSelection?: InitialSelectionType
  displayMode?: ResponsiveValuesTypes.ResponsiveValue<DisplayMode>
  sliderDisplayThreshold?: number
  sliderArrowSize?: number
  sliderItemsPerPage?: ResponsiveValuesTypes.ResponsiveValue<number>
  /** Used to override default CSS handles */
  classes?: CssHandlesTypes.CustomClasses<typeof SKU_SELECTOR_CSS_HANDLES>
}

**Comment by Luis-eplus** [comment]
hey, Sorry to keep you waiting, this bug was resolved in v0.2.20

thanks for your report
